pkg/controller/daemon/daemon_controller.go

```diff
@@ -1472,8 +1482,11 @@ func (o podByCreationTimestamp) Len() int      { return len(o) }
 func (o podByCreationTimestamp) Swap(i, j int) { o[i], o[j] = o[j], o[i] }
 
 func (o podByCreationTimestamp) Less(i, j int) bool {
-	if o[i].CreationTimestamp.Equal(&o[j].CreationTimestamp) {
-		return o[i].Name < o[j].Name
+	// Early CreationTimestamp first
```

Comment:
Since you are not changing the logic here, please revert this function to what it was before to minimize debugging surface in case any issue is found later.

Reply:
Done, reverted it :)